Jellyfin.Server.Implementations/Users/UserManager.cs

```diff
@@ -167,6 +166,7 @@ public void UpdateUser(User user)
         {
             using var dbContext = _dbProvider.CreateContext();
             dbContext.Users.Update(user);
+            _users[user.Id] = user;
```

Comment:
I kind of assumed this wasn't necessary. The user objects that are modified, aren't they already in the dict?